libbpfgo.go

```diff
@@ -776,6 +776,15 @@ func (b *BPFMap) GetValueReadInto(key unsafe.Pointer, value *[]byte) error {
 	return nil
 }
 
+func (b *BPFMap) SetInitialValue(value unsafe.Pointer) error {
+	sz := b.ValueSize()
+	ret := C.bpf_map__set_initial_value(b.bpfMap, value, C.ulong(sz))
```

Comment:
Is this the intended use case of this function? How is using this different from `bpf_map__update_elem`? It would be good contribute the docs to `libbpf.readthedocs.org` for it.

Reply:
> Is this the intended use case of this function?

Yes, this function can be used to update internal maps: https://lore.kernel.org/bpf/87fszrkhi7.fsf@toke.dk/

> How is using this different from `bpf_map__update_elem`

* `bpf_map__update_elem` update map via `bpf` syscall and only support public maps.
* `bpf_map__set_initial_value` update map via `memcpy` and only support internal maps.